### PR TITLE
create the contact section of the landing page #1

### PR DESCRIPTION
**Pull request by hamzah-bahar**
### Contact Section  

- The original contact (big screen) 
image

- My contact:  
image

- Original (medium size screen):
image

- Mine:
image

**And small screens:** 

- Them: 

image

- Me:

image

- Small screen is the default. 
- Medium size screen is at min-width: 780px 
- Big size screen is at min width: 1000px   

colors used: 
image
for icons, buttons, lines and the focus state of the input boxes. 

image
image
for headers and paragraphs

### CODE REVIEW

**Review by hshahwan — APPROVED** [review]

**Review by Rahaf-96 — APPROVED** [review]
Looks Great!

**Review by bayan-404 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -10,6 +10,56 @@
...
+    
+                    </div>
+    
+                </div>
+                
+                <div class="right-section">
+                    <form class="contact-method">
+                        <div class="input-method name-email">
+                            <input  type="text" placeholder="Your Name">
+                            <input  type="text" placeholder="Your Email">
```
Comment (bayan-404) [inline index.html]: there's a `type = "email"` that you can use 
it's more accessible this way

File: index.html [hunk 2]
```diff
@@ -10,6 +10,56 @@
...
+                        </div>
+                        <input class="mb-1 input-method" type="text" placeholder="Subject">
+                        <textarea class="mb-1 input-method" cols="30" rows="8" placeholder="Message"></textarea>
+                        <button>Send Message</button>
```
Comment (bayan-404) [inline index.html]: you can add `type="submit"` to the form button 
it defines the button that submits the form data 
(more accessible, and you will need it when you want to store your data into a form-handler)

**Review by bayan-404 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (hamzah-bahar) [reply index.html]: Of course there is, but at this point I don't care about what will happen behind the scenes (backend). This is the main reason why I didn't pay attention to those specifics.  
Thank you for the note

File: index.html (hunk 2 above)
Comment (hamzah-bahar) [reply index.html]: Again, I don't care about submitting the form, or else I would've added both action and method attributes to the form itself.

File: index.html (hunk 1 above)
Comment (bayan-404) [reply index.html]: it's always up to you 
but I want to clarify that it's not only a matter of backend or server-side
it's also very important to consider creating accessible websites 
I suggest you read more about accessibility and why it's needed 
https://www.w3.org/WAI/fundamentals/accessibility-intro/#:~:text=Web%20accessibility%20means%20that%20websites,contribute%20to%20the%20Web

Comment (hamzah-bahar) [reply index.html]: Thank you. I appreciate it 👍

**Pull request merged**